### ISSUE TITLE
Are controlled vocabularies time sensitive?

### ISSUE THREAD

**Issue by pzermoglio** [tdwg/dwc-qa#65]
(These questions were posed during the second chapter of the DwC Hour, 7 Mar 2017).

What happens when a controlled vocabulary, let us say "Music" is valid for 10 years, let's say, but then the working group or its equivalent decides to split into two, e.g., "Instrumental music" and "Voice music". Thus a content before a certain date is different than after. So, let us say DwC adds to PreservedSpecimen "HebariumSheet" and "PinnedSpecimen". Are such types of splits acceptable?
In other words, what happens when distinctions are introduced in a given field after a certain date?
Does it ever occur that a new controlled vocab term is introduced that splits or further defines already existing terms and thus terms in the controlled vocab are "time sensitive"?

**Comment by garymotz** [comment]
Could be addressed using suggestions in issue #69

**Comment by ansell** [comment]
There may have already been such a split from ``Observation`` (in some cases abbreviated down to ``O``) to ``HumanObservation`` and ``MachineObservation``.
